### PR TITLE
Fixing destroy_nodes , none platform deploy_node and adding console to none platform vms

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: scripts/install_environment.sh [hunk 1]
```diff
@@ -98,7 +98,7 @@ function install_runtime_container() {
 
 function install_packages() {
     echo "Installing dnf packages"
-    sudo dnf install -y make python3 python3-pip git jq bash-completion xinetd
+    sudo dnf install -y make python3 python3-pip git jq bash-completion xinetd aicli
```
Comment (YuviGold) [inline scripts/install_environment.sh:101]: ?
I think it's a python package.

File: terraform_files/none/consolemodel.xsl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
```
Comment (YuviGold) [inline terraform_files/none/consolemodel.xsl:1]: What does it do? Please elaborate on what's the benefit from adding this feature as part of the PR description

File: scripts/install_environment.sh (hunk 1 above)
Comment (tsorya) [reply scripts/install_environment.sh:101]: oops, you are right

File: terraform_files/none/consolemodel.xsl (hunk 2 above)
Comment (tsorya) [reply terraform_files/none/consolemodel.xsl:1]: there is nothing new , it is just copy paste from BM platform functionality

**Review by YuviGold — APPROVED** [review]

**Comment by tsorya** [comment]
/test e2e-metal-assisted-none
--


<br class="Apple-interchange-newline">

**Pull request merged**